test/unit/api/index.js

```diff
@@ -576,10 +576,6 @@ describe('api/index.js', () => {
         expect(() => api.userActivityReport('user', 'from', moment())).to.throw(TypeError);
         expect(() => api.userActivityReport('user', moment(), 'to')).to.throw(TypeError);
       });
-      it('should throw a ReferenceError if either parameter is not a valid date object', () => {
-        expect(() => api.userActivityReport('user', moment('from'), moment())).to.throw(RangeError);
-        expect(() => api.userActivityReport('user', moment(), moment('2017-02-29'))).to.throw(RangeError);
-      });
```

Comment:
I think this is useful. Why do you think it should go?

Reply:
Because we're not testing the ability of the error handling ability of the `api.userActivityReport` method, it doesn't get that far, instead we're only testing the ability of `moment` to handle dodgy input, which it does and throws the error before the `api.userActivityReport` method is ever even called!

Reply:
I think that's true for the first line of the test but not the second. So only the second line should go.

Here's some output from the REPL to show what I mean:
```js
$ node
> const moment = require('moment');
undefined
> let foo = moment('2017-02-29');
undefined
> foo.isValid()
false
> foo
moment.invalid(/* 2017-02-29 */)
```
Though the second test suffers from another problem which is that even if the date was valid it would be in the past.

Reply:
Ok, so moment still creates the dodgy dates as moment objects, which means *both* tests are still valid, but yes a proper date should be set for the `from` date in the second part...